src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/charts-and-dashboards-with-nrql.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: "Charts and dashboards: visualizing your data"
+tags:
+  - Visualize your data
+  - 'NRQL: New Relic query language'
+  - Get started
+translate:
+  - jp
+  - kr
+metaDescription: "Learn the basics of charts and dashboards, and how they relate to NRQL."
+redirects:
+---
+import dashboardsDashboardUi from 'images/dashboards_screenshot-crop_dashboard-ui.png'
+
+import chartsChartUi from 'images/queries-nrql_screenshot-crop_dashboard-chart-color.png'
+
+Queries help you find the information you need, and your charts and graphs based on those queries make the data useful to your organization. Charts and dashboards are the key to data visualization, giving you multiple options that enable you to create visually appealing and easily understandable data streams. You can make the most of your data by understanding the ways that NRQL queries lead in to charts and dashboards.
+
+## Charts [#charts]
+
+<img
+  title="Chart screenshot.png"
+  alt="A screenshot example of a chart"
+  src={chartsChartUi}
+/>
+<figcaption>
+  This is a chart created by a NRQL query showing the total number of transactions for each reported application across a period of 1 hour.
+</figcaption>
+
+Configuring a chart to represent your data is the last step of creating a NRQL query. New Relic offers [multiple options](/docs/query-your-data/explore-query-data/use-charts/chart-types) to help you find the perfect way to visualize your data, and you can do many things with your charts once you've made them. After you write your NRQL query, you can use charts to:
```

Comment:
Flesh out the process for creating a chart after NRQL. Ie: After writing the query and then you need to create a chart and save it.